packages/react/src/analytics/integrations/GTM/eventsMapper.ts

```diff
@@ -88,6 +88,8 @@ const eventsMapper: EventMappers = {
     return {
       products: get(properties, 'products', []).map(getProductData),
       orderId: properties.orderId,
+      total: properties.total,
```

Comment:
What is supposed to be the difference between `total` and `value` in checkout events ?

Reply:
I look into other integrations, and I saw that we are using total instead of value, but in some cases, we already have value mapped, such in EventTypes.PLACE_ORDER_STARTED.

To avoid breaking changes, and by according with our documentation, the value of grandTotal in checkout, would be fill the "Total" in analytics contract. So in this case, I maintain this two options, only to maintain compatibility.